packages/ui/constants/index.ts

```diff
@@ -39,6 +39,7 @@ export const ADMIN_FEE = {
 
 export const POOLS_PER_PAGE = 6;
 
+// TODO: We should replace this with NATIVE_DECIMALS from the @midas-capital/chains package
```

Comment:
I suppose all EVMs are like this!

Reply:
Good point. Not sure if EVM compatibility enforces native assets to have 18 decimals but could be!